ios/Runner/Info.plist

```diff
@@ -41,5 +41,11 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>App needs access to your location to start the hike</string>
```

Comment:
How long can this text be? Can we have something like "Your location is needed to guide you during the hike and show your location to the group" ?

Reply:
Sure i'll change it.